help/C/index.docbook

```diff
@@ -1404,6 +1363,62 @@
                     Select this option to display the command line that was used to start the process, including arguments.
                   </para>
                 </listitem>
+                <listitem>
+                  <para>
+                    <guilabel>Waiting Channel</guilabel>
+                  </para>
+                  <para>
+                    Select this option to display the waiting channel of a process where it is waiting for a resource.
```

Comment:
Select this option to display the waiting channel, whereby a process waits for a resource.